ext/crow/crow/socket_adaptors.h

```diff
@@ -94,7 +99,7 @@ namespace crow
 
         boost::asio::io_service& get_io_service()
         {
-            return raw_socket().get_io_service();
+            return GET_IO_SERVICE(raw_socket());
```

Comment:
GET_IO_SERVICE

Reply:
Crow is no longer maintained. So in some future version of the explorer it will be replaced with something that actually is alive.